src/upnp.cpp

```diff
@@ -1106,7 +1109,7 @@ void find_error_code(int const type, string_view string, error_code_parse_state&
 	}
 	else if (type == xml_string && state.in_error_code)
 	{
-		std::string error_code_str(string.begin(), string.end());
+		std::string const error_code_str(string.begin(), string.end());
 		state.error_code = std::atoi(error_code_str.c_str());
```

Comment:
the ``error_code_str`` variable could be removed entirely here by just calling: ``string.to_string().c_str()`` and pass that into ``std::atoi()`` directly.